src/main/fc/config.c

```diff
@@ -242,6 +246,12 @@ static void validateAndFixConfig(void)
 
     validateAndFixGyroConfig();
 
+    buildAlignmentFromStandardAlignment(&compassConfigMutable()->mag_customAlignment, compassConfig()->mag_alignment);
```

Comment:
What is the purpose of doing this?

Reply:
this is the config transition code that sets the custom values from the enum. we talked about it in slack.

Reply:
Ah yes.